source/index.html.md.erb

```diff
@@ -28,3 +28,41 @@ the [Prison API](https://api-dev.prison.service.justice.gov.uk/swagger-ui/index.
 
 [![Container diagram](https://raw.githubusercontent.com/ministryofjustice/hmpps-integration-api/main/docs/diagrams/container.svg)](https://raw.githubusercontent.com/ministryofjustice/hmpps-integration-api/main/docs/diagrams/container.svg)
 
+## API changes and versioning
+
+As HMPPS Integration API evolves, more features will be added, and features that already exist will change. Both consumers and the MoJ must be prepared to handle these changes, and to manage them carefully.
```

Comment:
```suggestion
As HMPPS Integration API evolves, more features will be added, and features that already exist will change. Both consumers and MOJ must be prepared to handle these changes, and to manage them carefully.
```

![image](https://user-images.githubusercontent.com/42817036/228812206-ff6f1d06-2a3b-4476-9237-247243648d77.png)

Don't worry, I only learnt this recently as well! Don't forget to update the usages below.

Reply:
whoa